### PR TITLE
create setup.py for flake8-config-4catalyzer

### PR DESCRIPTION
**Pull request by cancan101**

### CODE REVIEW

File: packages/flake8-config-4catalyzer/setup.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+from setuptools import setup
+
+
+setup(
+    name='flake8-config-4catalyzer',
+    version='0.1.0',
+    url='https://github.com/4Catalyzer/python/blob/packages/flake8-config-4catalyzer',
```
Comment (taion) [inline packages/flake8-config-4catalyzer/setup.py]: `tree`

**Review by taion — APPROVED** [review]
make sure this is build-able. good to go if so. please publish as well.

**Comment by itajaja** [comment]
nice!

**Comment by itajaja** [comment]
how do you use this?

**Pull request merged**